code/modules/surgery/helpers.dm

```diff
@@ -81,7 +81,26 @@
 			return 1
 	return 0
 
-
+/proc/get_pain_modifier(mob/M) //returns modfier to make surgery harder if patient is conscious and feels pain
```

Comment:
Can you typecast this to `mob/living/carbon/human`?

Reply:
I could, yeah. It gets typecast in the proc itself as is. I tried to make it the same as the other helper proc that gets called in surgery, 'get_location_modifier(mob/M)'.

I don't think there should be a big difference between doing it here or in the proc, so it's mostly a style thing. What do others think?

Reply:
I agree with @Citinited

Reply:
Okay, changed it.